README.md

```diff
@@ -72,16 +72,20 @@ It is important that you do not use the same `DEPLOY_ENV` for both build and dep
 
 ### Deploy
 
-Create Concourse Lite with `make`. There are targets to select the target AWS account, and to select the profiles to apply. For instance for a DEV build concourse bootstrap:
+Create Concourse Lite with `make`. There are Make targets to select the target AWS account, and to select the profiles to apply.
+
+You can create two different things: a deployer concourse or a build concourse.
+
+To start deploying a new Cloud Foundry environment:
 
 ```
-make dev build-concourse bootstrap
+make dev deployer-concourse bootstrap
 ```
 
-Or a DEV deployer concourse bootstrap:
+Or to start deploying a new build environment:
```

Comment:
I think it'd be good to explain (briefly) why/when you'd want either a build and/or deployer concourse in that section.

Reply:
This is a good change, I agree with Andy, here is a rushed suggestion

```suggestion

The above command will deploy a Deployer Concourse which is used for deploying a PaaS.

It is possible to deploy a Build Concourse instead. A Build Concourse is responsible for building and versioning releases, as well as deploying manuals, documentation, product pages, and more.
  
To start deploying a new build environment:
```